public/js/term-highlight.js

```diff
@@ -0,0 +1,34 @@
+jQuery(function($) {
+  $(document).ready(function () {
+    var searchTerm = $('h1.sr-only').data('term').toLowerCase();
+        // remove speach mark
+        searchTerm = searchTerm.replace(/[".]/g,'');
+        // remove special characters
+        searchTerm = searchTerm.replace(/[;]/g,'');
+        // Change Doctor to Dr
+        searchTerm = searchTerm.replace('doctor','dr');
+
+    var terms = searchTerm.split(/\W+/);
+
+    for (var i = 0, len = terms.length; i < len-1; i++) {
+      terms.push(terms[i]+' '+terms[i+1]);
+    }
+
+    if ($.inArray( searchTerm, terms ) === -1){
```

Comment:
Re: server side, an issue has been raised for this: https://github.com/nhsuk/gp-finder/issues/106. Is this necessary to be done as part of this PR?